src/main/java/com/global/api/builders/ManagementBuilder.java

```diff
@@ -152,6 +154,13 @@ public BigDecimal getTotalCredits() {
     public BigDecimal getTotalDebits() {
         return totalDebits;
     }
+    
+    public Integer getEstimatedNumTxn(){
+        return estimatedNumTxn;
+    }
+    public boolean getIsFinal(){
+        return isFinal;
+    }
```

Comment:
If we update the management builder to use a single method, we'll probably want change these to match.